### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by pachia-lee**
- Made UI changes for login screen
- Made UI changes for home screen

### CODE REVIEW

File: Resources/.DS_Store
Comment (benderje26) [inline Resources/.DS_Store:1]: Could be wrong, but I think this is like the mac folder info thing? I figure you probably didn't mean to include it in the commit

**Review by benderje26 — COMMENTED** [review]
Just fix the .DS_Store. Otherwise, this looks great, Pachia! Thank you!

File: Resources/.DS_Store
Comment (pachia-lee) [reply Resources/.DS_Store:1]: I'm not sure why it's there, but I didn't mean to include it in the commit..

Comment (pachia-lee) [reply Resources/.DS_Store:1]: I think we might need to use this for the .gitignore instead:
https://github.com/dotnet/maui/blob/main/.gitignore

**Review by wangj97 — APPROVED** [review]
:D

**Pull request merged**